package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ilp-routing",
-  "version": "5.3.0",
+  "version": "6.0.0",
```

Comment:
Bumping the version should be done separately (after the PR is merged). i.e. `npm version major` -- that will automatically create a tag as well.